starter-code/public/stylesheets/style.css

```diff
@@ -1,8 +1,35 @@
 body {
```

Comment:
loved the personalisation and the styles of the page!!! L O V E D  I T !!! GREAT JOB 🥇